assets/app/styles/_core.less

```diff
@@ -93,6 +93,15 @@
    .flex(@columns: 0 0 55px);
    text-align: right;
  }
+ .ui-select-bootstrap {
+   .ui-select-placeholder {
+     font-weight: normal;
+   }
+   .ui-select-toggle > .caret {
+    font-style: normal;
+    margin-top: -5px;
+   }
+ }
```

Comment:
Wondering if we shouldn't just make this is the default style for ui-select everywhere instead of only in the data-toolbar.